f2/src/utils/yupSchema.js

```diff
@@ -5,10 +5,13 @@ import { Trans } from '@lingui/macro'
 
 export const yupSchema = () => {
   return {
-    phoneSchema: Yup.string().matches(regexDef().phoneRegExp, {
-      excludeEmptyString: true,
-      message: 'Please enter a valid phone number',
-    }),
+    phoneSchema: Yup.string()
+      .transform((value) => value.replace(/[\s()+-.]|ext/gi, ''))
```

Comment:
Why are we removing the phone number regex?

Reply:
Phone number validation is broken now, "++--" is accepted as a valid phone number.

Reply:
As discussed in team stand-up, we need to populate this new international number rule to the other place where we use phone validation.  (How did incident start?)  Since business side has such needs where many scam/crime calls show international number. Or what could do is keep the old phone schema in the code for future use, while change the name of new international number to another name.  But from clean code perspective, we can just comment out phoneSchema (old), and add comments on new one if we wish to keep old validation for reference

Reply:
It is fine if we decide to modify what is valid phone number but this validation is broken. If we need to validate any type of phone number maybe it would be easier to use a node package like [phone](https://www.npmjs.com/package/phone).

Reply:
Phone validation is not broken, this PR just lets Phone number in Contact form and (How did incident start) form follow the same rule that allows more loose validation since we allow International Number. The initiative for this request is to lower the validation bar when user types in phone number (Now we states international number to be allowed), so that we can reduce the not-completion rate when user fills the form.  For international number, Min. is 7, while E164 defines Max number either 12 or 15, it will add more complexity which may ends up adding (pick Country Code drop down menu, save a large number of valid country code etc). From what I heard from business team is to lower the validation bar for now until they have strong desire to let user to confirm country code.

Reply:
If this is the desired outcome why validate at all?
![phone](https://user-images.githubusercontent.com/62246403/96285887-8a5c5d80-0fad-11eb-99d9-fd0f4ac68c13.png)

Reply:
I just added one validation to limit the Max number of valid digits to 15

Reply:
"++++" needs to be fixed.

Reply:
Issue of "++++" like allowed charactors Only (with no digits) pass validation has been resolved in the new  commit

Reply:
Validation still needs work. What is the number of max digits - 15 for phone number, 10 for extension? See image below.
![phone](https://user-images.githubusercontent.com/62246403/97005965-22f26080-150d-11eb-9347-1c573f042a5e.png)

Reply:
As for Extension, I added limitation to allowing up to 8 digits o valid number [0-9], it is already in latest push and commit.
As for Phone Number, as discussed in standup, we had better not put too much strict rule on International Number which will largely increase non-completion rate.  This app is NOT any business web site which involves financial transaction, not other government web site like CRA which has to enforce much stricter validation rule on any input from user since this involve real money and session is not URGENT.  Our app is more like 911 APP or work flow,  heard many stories of over strict/over engineered rule or validation may put user into desperate or live/dead situation or even revealing such experience on public media or news.

Reply:
Extension does allow more than 8 characters.
![extension](https://user-images.githubusercontent.com/62246403/97020533-a7e67580-151f-11eb-8a1b-37bb12074b3f.png)

Reply:
One regex was not included in previous commit, now everything is in git

![image](https://user-images.githubusercontent.com/5130380/97021174-7b7f2900-1520-11eb-8518-cc63cf253ba7.png)